### DISCUSSION TITLE
Config option for shorter links after User ID

### DISCUSSION THREAD

**Discussion by Zawaken** [SergiX44/XBackBone#359] [Ideas]
**Is your feature request related to a problem? Please describe.**
The link given from XBackBone is quite long, not really a huge problem, but it makes it harder to type out from a mobile device to a computer or when a clipboard can't be used. For example in a tty.

**Describe the solution you'd like**
I'd like to be able to set how many characters the files in the userid folder gets assigned, so that the link looks better and is easier to type out when you can't use the clipboard.

**Additional context**
E.g. https://i.example.com/userid/abcd.png instead of https://i.example.com/userid/abcdefghjk.png 
abcd.png instead of abcdefghjk.png in the userid folder

**Comment by sergix44** [comment]
Currently the links are of fixed length to have a sufficient number of characters to make the enumeration of uploaded media less simple.
Instead of shortening the urls, we could display a text field with the url that can be copied from mobile too

**Comment by Zawaken** [comment]
So if I understand correctly, the characters after the userid are 10 characters to help against bruteforcing of the uploaded media?
If so, that is a valid reason for having a fixed length,
But what I am asking for is not really shortening the links for everyone, I'd just want an option that can be set in a config file or on a per user basis in the application, that defaults to what it is now (10).

And copying the url from a text field on mobile wouldn't necessarily make what I put as the problem any easier. as I still would have to type it into the url bar of my browser on my computer (or I could just log in to my XBackBone account.), or into the tty where I have no viable way of logging in to XBackBone.

**Comment by rossiscool123** [comment]
You can change the URL length in  "/app/helpers.php" Search for "**function humanRandomString**" and change the **$length** this will be applied to every user though.
@Zawaken

**Comment by Zawaken** [comment]
Thank you @rossiscool123, this works exactly the way I want it to, as I am the only current user of my XBackBone instance, the fact that it changes it for all users is not an issue.
Though I am using docker to deploy XBackBone, so I would assume that this will be changed back whenever the container gets restarted. 
But it is probably possible to change this so that it persists.
